frontend/src/pages/BasePage.js

```
@@ -0,0 +1,395 @@
+    } else if (Array.isArray(children)) {
+      children.forEach(child => {
+        if (typeof child === 'string') {
+          element.appendChild(document.createTextNode(child));
+        } else if (child instanceof HTMLElement) {
+          element.appendChild(child);
+        }
+      });
+    }
+
+    return element;
+  }
```

Biome (2.1.2)

[error] Duplicate class member name "createElement"



(lint/suspicious/noDuplicateClassMembers)